CHANGELOG.md

```diff
@@ -4,6 +4,48 @@
 
 ### New features
 
+#### Check checkboxes by using the `checkedValues` option
```

Comment:
I've opted to document this as three separate (but very similar) features because the words (checked / selected) and the options (`checkedValues` / `checkedValue` / `selectedValue`) are different in each case and trying to write it is a single feature got messy quickly.

Reply:
Makes sense!

Can see from line 35 that 'option' is a bit of a problem term in this content...

Wonder could we replace 'option' with 'parameter' throughout, whenever we refer to `checkedValues`, etc.? Or would we not want to do that, because of the PR a few days ago that made 'option' the word we use for these things?